common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -0,0 +1,1326 @@
+		name = holy_site_silvermoon_effect_name
+		archers_damage_mult = 0.25
+		forest_advantage = 7
+	}
+}
+sunstrider_isle = {
+	county = c_sunstrider_isle
+	
+	character_modifier = {
+		name = holy_site_sunstrider_isle_effect_name
+		development_growth_factor = 0.1
+		theocracy_government_vassal_opinion = 10
+	}
+}
+andaroth = {
+	county = c_andaroth
+	
+	character_modifier = {
+		name = holy_site_andaroth_effect_name
+		stewardship = 1
+		monthly_piety_gain_mult = 0.05
+	}
+}
+#dalaran
+#################
+# deathbringer
+#################
+icecrown = {
+	county = c_icecrown
+	
+	character_modifier = {
+		name = holy_site_icecrown_effect_name
+		men_at_arms_limit = 1
+		men_at_arms_cap = 2
+	}
+}
+draktharon = {
+	county = c_draktharon
+	
+	character_modifier = {
+		name = holy_site_draktharon_effect_name
+		dread_gain_mult = 0.3
+		archers_pursuit_mult = 0.1
+	}
+}
+#lordaeron
+#sunwell_plateau
+#dalaran
+
+####################
+# druidism_group
+####################
+boralus = {
+	county = c_boralus
+	
+	character_modifier = {
+		name = holy_site_boralus_effect_name
+		embarkation_cost_mult = -0.2
+		naval_movement_speed_mult = 0.2
+	}
+}
+alterac = {
+	county = c_alterac
+	
+	character_modifier = {
+		name = holy_site_alterac_effect_name
+		mountains_advantage = 8
+		skirmishers_screen_add = 4
+	}
+}
+#lordaeron
+#stromgarde
+#gilneas
+
+### wolf_cult_religion
+blackwald = {
+	county = c_blackwald
+	
+	character_modifier = {
+		name = holy_site_blackwald_effect_name
+		farmlands_advantage = 7
+		stewardship = 1
+		plains_advantage = 6
+	}
+}
+xoojith = {
+	county = c_xoojith
+	
+	character_modifier = {
+		name = holy_site_xoojith_effect_name
+		religious_vassal_opinion = 10
+	}
+}
+#Shrine of Goldrinn
+jillaei = {
+	county = c_jillaei
+	
+	character_modifier = {
+		name = holy_site_timbermaw_effect_name
+		prowess = 2
+		enemy_hard_casualty_modifier = 0.1
+	}
+}
+darnassus = {
+	county = c_darnassus
+	
+	character_modifier = {
+		name = holy_site_darnassus_effect_name
+		forest_cancel_negative_supply = yes
+		church_holding_holding_build_gold_cost = -0.3
+	}
+}
+#nordrassil
+
+### ursine
+#nordrassil
+grizzlemaw = {
+	county = c_grizzlemaw
+	
+	character_modifier = {
+		name = holy_site_grizzlemaw_effect_name
+		same_faith_opinion = 5
+		taiga_advantage = 5
+	}
+}
+emberglade = {
+	county = c_emberglade
+	
+	character_modifier = {
+		name = holy_site_grizzlemaw_effect_name
+		dynasty_opinion = 15
+	}
+}
+timbermaw = {
+	county = c_timbermaw
+	
+	character_modifier = {
+		name = holy_site_timbermaw_effect_name
+		martial_per_piety_level = 1
+		garrison_size = 0.1
+	}
+}
+blackmaw = {
+	county = c_blackmaw
+	
+	character_modifier = {
+		name = holy_site_blackmaw_effect_name
+		prowess_per_stress_level = 5
+		learning = 1
+	}
+}
+
+### kaldorei_religion
+#nordrassil
+#darnassus
+astranaar = {
+	county = c_astranaar
+	
+	character_modifier = {
+		name = holy_site_astranaar_effect_name
+		stewardship = 2
+		faith_creation_piety_cost_mult = 0.3
```

Comment:
It's weird debuff.
So, player need to lose this province if they want to reform religion?

Btw, if this is mistake, then -30% reformation cost is the most weird buff I have seen. I suggest you to replace it with something else, otherwise conquering this holy site will make faith more venerable to new heresies.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99657934-d9dfee80-2a6f-11eb-8646-4579fc0f09fd.png)

</details>

Reply:
> Btw, if this is mistake, then -30% reformation cost is the most weird buff I have seen. I suggest you to replace it with something else, otherwise conquering this holy site will make faith more vulnerable to new heresies.

I don't agree. It makes religion more flexible. A very good buff if the religion is planned to be reformed. It will allow you to pick stronger tenets and doctrines for a lower price.

Reply:
Made it negative in 7f4a35af65c3e0a323ee5b05a28ead053b5ed94f.

Reply:
> It makes religion more flexible.

> It will allow you to pick stronger tenets and doctrines for a lower price.

I do not disagree with that. In fact, you just said the same thing as me, except you tried to make it sounds like a good buff, which is it not, if you don't plan to create a new heresy.

> A very good buff if the religion is planned to be reformed.

Only NE and their heresies will get this buff, since only they use this holy site.

So, the plan is to make Elunic religion the easiest to make heresies from?

Reply:
> I do not disagree with that. In fact, you just said the same thing as me, except you tried to make it sounds like a good buff, which is it not, if you don't plan to create a new heresy.

It's a great buff, but it's situative. I would like to have it in 75% of my playthroughs.

> So, the plan is to make Elunic religion the easiest to make heresies from?

I think so. It makes sense since the night elves changed a lot after the Third War and the War of the Ancients

Reply:
> It's a great buff, but it's situative. I would like to have it in 75% of my playthroughs.

Idk, personally, I wouldn't want for any characters of my faith to make new heresies. Especially when my character makes custom religion and tries to spread it as much as possible. Good thing that AI doesn't do it, I guess?

> I think so. It makes sense since the night elves changed a lot after the Third War and the War of the Ancients

How much, tho?
They outlawed arcane magic after the War of the Ancients (So, in-game terms, they changed one criminal doctrine).
Aren't they just allowed males to be priests and females to be druids after the Third War (So, in-game it will only change female priesthood doctrine to equal)?
Changing 2 doctrines in 10 000 years doesn't sound like "changed a lot", tbh.

Doctrines can be changed via events and decisions, like how Christian denominations can lose or gain ecumenism doctrine in vanilla, so I don't think it's a good idea to suggest players make a whole new religion instead of giving them a decision to change just one doctrine.

Reply:
> Idk, personally, I wouldn't want for any characters of my faith to make new heresies. Especially when my character makes custom religion and tries to spread it as much as possible. Good thing that AI doesn't do it, I guess?

Convert them back or size titles (if you really need). If you fear heresies, you shouldn't do holy wars and be wide. That's the main things that push heresies. Of course, you can punish people for picking Scholar focus, but I don't think it's worth it.
Basically, don't think it's bad to have a few religions in your realm especially if they're close. Together you can holy war everyone more often because you don't share the same fervour.

> How much, tho?
> They outlawed arcane magic after the War of the Ancients (So, in-game terms, they changed one criminal doctrine).

And allowed it back around Cata.

> Aren't they just allowed males to be priests and females to be druids after the Third War (So, in-game it will only change female priesthood doctrine to equal)?

> Changing 2 doctrines in 10 000 years doesn't sound like "changed a lot", tbh.

Before the Third War, they may have Fundamentalism doctrine. They didn't allow other races in their society and were isolationists.

Before the War of the Ancients, they may have no Sanctity of Nature tenet (they rediscovered Cenarius when Malfurion became the first druid). Also, they may be a different faith of Elunism on the Broken Isles without this tenet (the nightborne), but this requires more research.